components/Collections/src/MDCCollectionViewController.m

```diff
@@ -434,12 +434,12 @@ - (BOOL)collectionView:(UICollectionView *)collectionView
 
 - (void)collectionView:(UICollectionView *)collectionView
     didSelectItemAtIndexPath:(NSIndexPath *)indexPath {
-  [self updateFooterInfoBarIfNecessary];
+  [_footerInfoBar showAnimated:YES];
 }
 
 - (void)collectionView:(UICollectionView *)collectionView
```

Comment:
Same comment as above. Let me know if need anything further or if I'm misunderstanding the request.